setup.cfg

```diff
@@ -1,72 +1,3 @@
-[metadata]
-name = rdmo
-version = attr: rdmo.VERSION
-description = RDMO is a tool to support the systematic planning, organisation and implementation of the data management throughout the course of a research project.
-long_description = file: README.md
-long_description_content_type = text/markdown
-url = https://github.com/rdmorganiser/rdmo
-author = RDMO Arbeitsgemeinschaft
-email = rdmo-team@listserv.dfn.de
-license = Apache-2.0
-classifiers =
-    # see https://pypi.org/classifiers/
-    Development Status :: 5 - Production/Stable
-    Environment :: Web Environment
-    Framework :: Django :: 2.2
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: Apache Software License
-    Operating System :: OS Independent
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-
-[options]
-python_requires = >=3.6
-packages = find:
-include_package_data = true
-install_requires =
-    coverage==5.5
-    defusedcsv~=2.0.0
-    defusedxml~=0.7.1
-    Django~=3.2.14
-    django-allauth~=0.51.0
-    django-cleanup~=6.0.0
-    django-compressor==4.0
-    django-extensions~=3.2.0
-    django-filter~=21.1
-    django-libsass==0.9
-    django-mathfilters~=1.0.0
-    django-mptt~=0.13.4
-    django-rest-swagger~=2.2.0
-    django-settings-export~=1.2.1
-    django-widget-tweaks~=1.4.8
-    djangorestframework~=3.12.4
-    drf-extensions~=0.7.0
-    iso8601~=1.0.2
-    jsonfield~=3.1.0
-    Markdown~=3.3.7
-    pypandoc==1.8.1
-    pytest~=6.2.4
-    pytest-cov~=2.12.1
-    pytest-django~=4.4.0
-    pytest-dotenv~=0.5.2
-    pytest-mock~=3.6.1
-    pytest-pythonpath~=0.7.3
-    rules==3.3
-
-[tool:pytest]
-DJANGO_SETTINGS_MODULE = testing.config.settings
-testpaths = rdmo
-python_files = test_*.py
-python_paths = testing/
-
-[isort]
-known_django=django
-sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
-
 [flake8]
```

Comment:
Can this be moved to pyproject.toml as well?

Reply:
Sadly no. Flake8 is one of the last projects, that do not work with pyproject.toml, https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations

There are a few options here:
- leave the flake8 settings in setup.cfg
- add a dedicated .flake8 config file
- add yet another dependency, something like https://pypi.org/project/Flake8-pyproject/
- ditch flake8 and use ruff instead

I would suggest using ruff -> it is faster, it also more or less replaces isort.

See
flake8 under https://beta.ruff.rs/docs/rules/ and https://beta.ruff.rs/docs/rules/#isort-i